README.md

```diff
@@ -42,12 +42,16 @@ ___
     - [Product page at iniLabs](https://inilabs.com/products/dynamic-vision-sensors/). [Buy a DVS](https://inilabs.com/buy/)
     - [Introductory videos about the DVS](https://inilabs.com/videos/dvs-introduction/)
     - [iniLabs](https://inilabs.com) invents, produces and sells neuromorphic technologies for *research*.
+- **DLS (Dynamic Line Sensor)**: Posch, C., Hofstaetter, M., Matolin, D., Vanstraelen, G., Schoen, P., Donath, N., and Litzenberger, M., *[A dual-line optical transient sensor with on-chip precision time-stamp generation](https://doi.org/10.1109/ISSCC.2007.373513)*, IEEE International Solid-State Circuits Conference - Digest of Technical Papers, Lisbon Falls, MN, US, 2007.
+    - [Fact sheet at AIT](https://www.ait.ac.at/fileadmin/mc/digital_safety_security/downloads/Factsheet_-_Linescan-Chip_DLS_en.pdf).
 - **DAVIS (Dynamic and Active-Pixel Vision Sensor)** :
 Brandli, C., Berner, R., Yang, M., Liu, S.-C., Delbruck, T., *[A 240x180 130 dB 3 µs Latency Global Shutter Spatiotemporal Vision Sensor](https://doi.org/10.1109/JSSC.2014.2342715)*, IEEE J. Solid-State Circuits, 49(10):2333-2341, 2014.
     - [Product page at iniLabs](https://inilabs.com/products/dynamic-and-active-pixel-vision-sensor/). [Buy a DAVIS](https://inilabs.com/buy/)
     - **Color-DAVIS**: Li, C., Brandli, C., Berner, R., Liu, H., Yang, M., Liu, S.-C., Delbruck, T., *[Design of an RGBW Color VGA Rolling and Global Shutter Dynamic and Active-Pixel Vision Sensor](https://doi.org/10.1109/ISCAS.2015.7168734)*, IEEE Int. Symp. Circuits and Systems (ISCAS), Lisbon, 2015, pp. 718-721.
 - **ATIS (Asynchronous Time-based Image Sensor)**: Posch, C., Matolin, D., Wohlgenannt, R. (2011). *[A QVGA 143 dB Dynamic Range Frame-Free PWM Image Sensor With Lossless Pixel-Level Video Compression and Time-Domain CDS](http://doi.org/10.1109/JSSC.2010.2085952)*, IEEE J. Solid-State Circuits, 46(1):259-275, 2011.
     - [Chronocam](http://www.chronocam.com/)
+    - [AIT Austrian Institute of Technology](https://www.ait.ac.at/themen/new-sensor-technologies/optical-sensor-systems-for-industrial-processes/)
+- **ATIS-L (ATIS Line Sensor)**: Belbachir, A., Schraml, S., Mayerhofer, M., and Hofstaetter, M., *[A novel HDR depth camera for real-time 3D 360-degree Panoramic vision](https://doi.org/10.1109/CVPRW.2014.69)*, IEEE Conf. Comput. CVPR Workshops, Jun. 2014, pp. 419-426.
```

Comment:
Since the reference "A Novel HDR Depth Camera for Real-time 3D 360-degree Panoramic Vision" already exists in the list, I suggest that you set a link to that reference, as in other examples, to avoid duplicating entries.

Reply:
The problem is that the ATIS-L is not really documented. I suggest under this circumstances to skip the whole entry ATIS-L